afdko/Tools/Programs/makeotf/makeotf_lib/source/hotconv/cmap.c

```diff
@@ -843,8 +902,6 @@ static Format4 *makeFormat4(cmapCtx h, unsigned long *length) {
         truncateCmap = 1;
 
 	while (notOK) {
-		unsigned int numOrdered = 0;
-		unsigned int numNotOrdered = 0;
 
 		fmt->glyphId.cnt = 0;
         if (h->mapping.cnt == 0)
```

Comment:
seems to be another mix of tabs vs spaces here, please make the indentation consistent